nifi-nar-bundles/nifi-beats-bundle/nifi-beats-processors/src/main/java/org/apache/nifi/processors/beats/ListenBeats.java

```diff
@@ -85,11 +87,21 @@
         .identifiesControllerService(RestrictedSSLContextService.class)
         .build();
 
+    public static final PropertyDescriptor CLIENT_AUTH = new PropertyDescriptor.Builder()
+        .name("Client Auth")
+        .displayName("Client Auth")
+        .description("The client authentication policy to use for the SSL Context. Only used if an SSL Context Service is provided.")
+        .required(false)
+        .allowableValues(RestrictedSSLContextService.ClientAuth.values())
+        .defaultValue(RestrictedSSLContextService.ClientAuth.REQUIRED.name())
+        .build();
+
     @Override
     protected List<PropertyDescriptor> getAdditionalProperties() {
         return Arrays.asList(
             MAX_CONNECTIONS,
-            SSL_CONTEXT_SERVICE
+            SSL_CONTEXT_SERVICE,
+	    CLIENT_AUTH
```

Comment:
Forgot to mention that, there were some lines containing TAB characters, that caused Travis CI build failure. I'll fix that when merging. Please ensure contrib-check pass successfully before submitting a PR next time. You can do so by a command like:
```
mvn verify -Pcontrib-check -f nifi-nar-bundles/nifi-beats-bundle/
```